exporter/exporter.go

```diff
@@ -12,7 +13,7 @@ type Exporter interface {
 
 type ExporterInstance interface {
 	Name() string
-	Export(ctx context.Context, src Source, sessionID string) (map[string]string, error)
+	Export(ctx context.Context, src Source, sessionID string) (*controlapi.ExporterResponse, error)
```

Comment:
Not a huge issue but ideally we don't use the proto types in here. Can be just array of maps.

Reply:
if you dive a little bit in the PR you will that sometimes I append `ExportersResponse` with `ExporterResponse` So for this I should keep type compatibility between `ExportersResponse` and `ExporterResponse`